component/attr_cache/attr_cache.go

```diff
@@ -802,7 +775,13 @@ func (ac *AttrCache) CopyFromFile(options internal.CopyFromFileOptions) error {
 		}
 		// TODO: Could we just update the size and mod time of the file here? Or can other attributes change here?
 		// TODO: we're RLocking the cache but we need to also lock this attr item because another thread could be reading this attr item
-		ac.invalidatePath(options.Name)
+
+		toBeInvalid, err := ac.cacheMap.get(options.Name) //empty for TestCopyFromFileDoesNotExist()
+		if err != nil {
```

Comment:
This error should never be returned to the calling function, so it should have a different name to make sure it doesn't get returned at the bottom.